tools/check_constitution.py

```
@@ -0,0 +1,385 @@
+
+        b_status, h_status = _status_line(base_text), _status_line(head)
+        if b_status is None or h_status is None:
+            # Never an append-only finding: a missing status line is a SHAPE defect,
+            # and reporting it here made the repair its own trigger — an entry that
+            # landed malformed could never be fixed and stayed CI-red forever.
+            findings.append(
+                f"entry-shape (§12): {path} has no `**Status:**` line — repair it before "
+                f"it lands; an accepted entry cannot be edited afterwards"
+            )
+            continue
+        strip = lambda t: [l for l in t.split("\n") if not STATUS_LINE.match(l)]  # noqa: E731
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)